src/status_im2/navigation/screens.cljs

```diff
@@ -116,7 +116,8 @@
      :component enable-notifications/enable-notifications}
 
     {:name      :sign-in
-     :component sign-in/view}
+     :component sign-in/view
+     :options   {:layout {:orientation ["portrait"]}}}
```

Comment:
portrait is the default option for all screens  so it shouldn't be set here, seem like somehow this setting was overwritten for this screen, by some overlay or ...

Reply:
Will investigate more